### ISSUE TITLE
Loading tiles are really bad, and obvious. You need to blend between the loaded tile and the next tile to be loaded.

### ISSUE THREAD

**Issue by rbyjoswald** [mapbox/mapbox-unity-sdk#1701]
Hello,

Is there a roadmap that shows that this issue is worked on, or not? It's been raised about 2 years ago, and so far has received a "not doable in short term"-answer and was then just closed. This issue is a deal-breaker for me. It's been reported to be a deal-breaker for others as well. It would be great to know if this is being worked on any time in the near future or not. A clear "no, we'll not fix that" will be a perfectly fine answer for me.

Regards,

Jürgen

**Comment by brnkhy** [comment]
@rbyjoswald I agree communication has been pretty bad for a while and you are right about how much it sucks not to have a proper public roadmap. I'm really hot happy about those either and I'm sorry for it.
At the moment I'm working on improving the performance on raster images. So imagery and elevation sections of the map. On parallel I'm working on `OfflineMaps` feature (https://github.com/mapbox/mapbox-unity-sdk/issues/1671). It's already on a branch but I still need to merge performance stuff with offline maps and I'm afraid that's still quite a bit of work.
